src/presentation/http/router/note.test.ts

```diff
@@ -10,17 +10,22 @@ describe('Note API', () => {
       const expectedStatus = 200;
       /* eslint-disable @typescript-eslint/naming-convention */
       const expectedNote = {
```

Comment:
naming is not clear enough now. I'd suggest to get rid of this variable and put an object directly in "expect" statement. It will better fit the "AAA" test best practice